docs/conventions/docs/rst.rst

```diff
@@ -15,10 +15,9 @@ ReStructuredText
 General
 =======
 
-With the exception of pre-formatted text (e.g., literals and code
-blocks), lines should not exceed 72 characters. At this width, GitHub
-pull request diffs will never be soft wrapped, which aids readability
-and collaboration.
+With the exception of pre-formatted text (e.g., literals and code blocks),
+lines should not exceed 72 characters. At this width, GitHub pull request diffs
```

Comment:
I think we use 79 in the CrateDB docs,  did this change at some point?

Reply:
Yeah it is 79 now. I suppose this section is outdated? Was it 72 at some prior point?

Reply:
> Was it 72 at some prior point?

I don't think so, at least not for the CrateDB docs.  Maybe Naomi knows more.